src/migrations/20240208123212-create-stat-traffic-usage-table.js

```diff
@@ -0,0 +1,24 @@
+
+exports.up = (db, callback) => {
+    db.runSql(
+        `
+        CREATE TABLE IF NOT EXISTS stat_traffic_usage(
+            day DATE NOT NULL,
+            traffic_group TEXT NOT NULL,
```

Comment:
ended up naming this traffic_group as the `group` name we initially came up with was a reserved keyword in Postgresql